task-list/app/views/tasks/index.html.erb

```diff
@@ -1,7 +1,18 @@
-<div class="tasks">
-  <ul>
+<table>
+  <col width="80%">
```

Comment:
It is generally better to add a class to these and then set the width within the CSS file instead of directly on this item